lib/common_components/change_atsign_bottom_sheet.dart

```diff
@@ -44,6 +46,17 @@ class _AtSignBottomSheetState extends State<AtSignBottomSheet> {
     setState(() {});
   }
 
+  Future<void> _checkForPermissionStatus() async {
+    final existingCameraStatus = await Permission.camera.status;
+    if (existingCameraStatus != PermissionStatus.granted) {
+      await Permission.camera.request();
+    }
+    final existingStorageStatus = await Permission.storage.status;
```

Comment:
@Pritam-deb I think you missed to remove this permission.

Reply:
done